events_webhook/src/main/java/com/adobe/aio/event/webhook/service/EventVerifier.java

```diff
@@ -41,19 +39,24 @@ public class EventVerifier {
 
```

Comment:
`@Service` / component annotation is required so that it can be auto injected in `eg-auditor`'s -> `EventServiceImpl` as here in this PR

Reply:
If this is the only reason that we need a @service/ @component annotation, I don't see it as a compelling reason to include the Spring framework dependency. 
Why can't we simply create an instance of `EventVerifier` in eg-auditor?